recipes/azure-core-cpp/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "azure-core-cpp" %}
+{% set version = "1.10.0" %}
+{% set sha256 = "e6ae26ad30376bef6272024564764ba7bf5756063dc94cc4b74180cd004ab75e" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/Azure/azure-sdk-for-cpp/archive/refs/tags/azure-core_{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
```

Comment:
This should definitely have a ru_exports as having it in host leads to a run dependency on it.

Reply:
Ok, I'll add